sources/web/datalab/polymer/modules/Utils.ts

```diff
@@ -112,4 +112,20 @@ class Utils {
     }
   }
 
+  /**
+   * Called when an API call fails, so that a connection error message can be shown.
+   */
+  static connectionFailed() {
```

Comment:
Are you planning on calling these methods from anywhere other than AppManager? It seems a little odd to bounce these calls through Utils.